maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/integration/MeansAssessmentIntegrationTest.java

```diff
@@ -264,4 +267,111 @@ void givenErrorCallingMaatApi_whenUpdateIsInvoked_thenServerErrorIsReturned() th
+        request.getApplicationDTO().getAssessmentDTO().getFinancialAssessmentDTO().getFull().getAssessmnentStatusDTO().setStatus(CurrentStatus.IN_PROGRESS.getStatus());
+        String requestBody = objectMapper.writeValueAsString(request);
+        String userSummaryResponse = objectMapper.writeValueAsString(MeansAssessmentDataBuilder.getUserSummaryDTOForPostProcess());
+        String repOrderDTO = objectMapper.writeValueAsString(MeansAssessmentDataBuilder.getRepOrderDTO());
+        String createIncomeEvidenceResponse = objectMapper.writeValueAsString(TestModelDataBuilder.getCreateIncomeEvidenceResponse());
+        String assessmentResponse = objectMapper.writeValueAsString(TestModelDataBuilder.getMaatApiAssessmentResponse());
+        String determineRepDecision = objectMapper.writeValueAsString(TestModelDataBuilder.getApiDetermineMagsRepDecisionResponse());
+
+
+        stubForOAuth();
+        wiremock.stubFor(post(urlMatching(CMA_URL))
+                .willReturn(WireMock.ok()
+                        .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
+                        .withBody(cmaResponse)));
+        stubForUpdateCrownCourtProceedings(ccpResponse);
+        stubForCalculateContributions(cccCalculateResponse);
+        stubForGetContributionsSummary(cccSummariesResponse);
+        stubForGetUserSummary(userSummaryResponse);
+        stubForGetRepOrders(repOrderDTO);
+        stubForUpdateSendToCCLF();
+        stubForCreateIncomeEvidence(createIncomeEvidenceResponse);
+        stubForUpdateFinancialAssessment(assessmentResponse);
+        stubForDetermineMsgRepDecision(determineRepDecision);
+        stubForATS();
+
+        stubForInvokeStoredProcedure(Scenario.STARTED, "DB_GET_APPLICATION_CORRESPONDENCE", maatApiResponse);
+        stubForInvokeStoredProcedure("DB_GET_APPLICATION_CORRESPONDENCE", "DB_ASSESSMENT_POST_PROCESSING_PART_1_C3", maatApiResponse);
+        stubForInvokeStoredProcedure("DB_ASSESSMENT_POST_PROCESSING_PART_1_C3", "DB_PRE_UPDATE_CC_APPLICATION", maatApiResponse);
+        stubForInvokeStoredProcedure("DB_PRE_UPDATE_CC_APPLICATION", "DB_ASSESSMENT_POST_PROCESSING_PART_2", maatApiResponse);
+        stubForInvokeStoredProcedure("DB_ASSESSMENT_POST_PROCESSING_PART_2", maatApiResponse);
+
+        mvc.perform(buildRequestGivenContent(HttpMethod.POST, requestBody, ENDPOINT_URL))
+                .andExpect(status().isOk());
+
+        assertStubForUpdateCrownCourtProceedings(1);
+        assertStubForInvokeStoredProcedure(1);
+        assertStubForHandleEformSerivce(1);
+        assertStubForDetermineMsgRepDecision(1);
+        assertStubForCreateIncomeEvidence(1);
+        assertStubForUpdateFinancialAssessment(1);
+        assertStubForCAT(1);
+    }
+
+    @Test
+    void givenAPostAssessmentProcessingEnabled_whenUpdateIsInvoked_thenAssessmentIsCreated() throws Exception {
```

Comment:
Presumably this should be `thenAssessmentIsUpdated`. Then again, not sure even that is suitable since the `then` is really focused on testing that the post processing ran correctly.

May also want to update the existing tests to check that the post processing didn't run. C3 is disabled in both of these tests, that will presumably also have to change.

There's also a rogue 'A' in both of the new test cases method names.